.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       - run:
           name: configure
-          command: ./configure --config dev --host dev.koding.com:8090 --hostname dev.koding.com --countlyApiPort "80"
+          command: ./configure --config dev --version $(git rev-parse --short HEAD) --host dev.koding.com:8090 --hostname dev.koding.com --countlyApiPort "80"
```

Comment:
what is the diff from [version = require('git-rev-sync').short()](https://github.com/koding/koding/blob/master/configure#L152) ?